Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript

```diff
@@ -2051,6 +2051,26 @@ function autoCompleteIFsAreBalanced pLineNumber, pScript
    end repeat
 
    put line 1 to tCounter of pScript into pScript
+   
+   local tWorkText, tOldDelim
+
+   -- remove quoted literals
+   put the itemDelimiter into tOldDelim
+   set the itemDelimiter to quote
+   repeat for each line aLine in pScript
+      if aLine contains quote then
+         repeat with i = 1 to the number of items of aLine
```

Comment:
I think this could be faster:
put 1 into i
repeat for each item tItem in aLine
 if i mod 2 = 0 then
  put empty into item i of aLine
 end if
 add 1 to i
end repeat

Reply:
...that was supposed to be a multiline comment for lines 2062 through 2066. I guess I don't have this multiline thing down yet.

Reply:
As you and I discovered when timing the two versions there was no discernible difference.
Thank you for your suggestion.